frappe/patches/v14_0/report_is_standard.py

```diff
@@ -0,0 +1,7 @@
+from frappe import qb
```

Comment:
qb is object attached to frappe. I don't think it can be... or rather _should be_ imported like this.